src/nvim/tui/tui.c

```diff
@@ -189,12 +191,46 @@ TUIData *tui_start(int *width, int *height, char **term)
                  100, 0);
 
   loop_poll_events(&main_loop, 1);
-  *width = tui->width;
-  *height = tui->height;
-  *term = tui->term;
   return tui;
 }
 
+void tui_attach(TUIData *tui)
```

Comment:
Ideally this should be in ui_client.c. still a separate function, but then taking the width, height and term name as parameters. (doesn't really matter for 0.9, but later on we do want to refactor ui_client.c to a reusable abstraction, which is why it is separate from tui.c in the first place)